### PR TITLE
BXMSDOC-1537:standalone controller

### PR DESCRIPTION
**Pull request by gsheldon**

### CODE REVIEW

File: docs/product-development-guide/src/main/asciidoc/chap-intelligent-process-server.adoc [hunk 1]
```diff
@@ -3723,6 +3723,57 @@ Allows you to start or stop a scanner that controls polling for updated containe
...
+CFG_KIE_USER::
+`org.kie.server.user`
+
+CFG_KIE_PASSWORD::
+`org.kie.server.pwd`
+
+CFG_KIE_TOKEN::
+`org.kie.server.token`
+
+The following properties are used when starting the {KIE_SERVER}. These properties direct the server to a {KIE_SERVER} controller (standalone or embedded), and the authentication values for the controller:
```
Comment (vidya-iyengar) [inline docs/product-development-guide/src/main/asciidoc/chap-intelligent-process-server.adoc]: Replace "a" with "an" in: "These properties direct the server to a {KIE_SERVER} ..." as '{KIE_SERVER}' is replaced with 'Intelligent Process Server'.

File: docs/product-development-guide/src/main/asciidoc/chap-intelligent-process-server.adoc [hunk 2]
```diff
@@ -3723,6 +3723,57 @@ Allows you to start or stop a scanner that controls polling for updated containe
...
+
+`org.kie.server.controller.templatefile`.
+
+By setting this property, the standalone controller will attempt to use that file for storing and retrieving container templates. If the property is not set, the following is used as a default file for container template storage:
+
+`System.getProperty("java.io.tmpdir")+System.getProperty("file.separator")+"template_store.xml"`
+
+[IMPORTANT]
+====
+The use of the default is not recommended, as it results in container templates being placed in a file that could be erased during the cleanup of a temporary directory.
```
Comment (vidya-iyengar) [inline docs/product-development-guide/src/main/asciidoc/chap-intelligent-process-server.adoc]: Is it "The use of the default **file** is not recommended..."?
"The use of the default is not recommended" sounds incomplete.

Comment (gsheldon) [reply docs/product-development-guide/src/main/asciidoc/chap-intelligent-process-server.adoc]: file. done.

File: docs/product-development-guide/src/main/asciidoc/chap-intelligent-process-server.adoc (hunk 1 above)
Comment (gsheldon) [reply docs/product-development-guide/src/main/asciidoc/chap-intelligent-process-server.adoc]: done

**Pull request closed without merging**